package.json

```diff
@@ -32,6 +32,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
+    "@headlessui/react": "^2.0.3",
```

Comment:
Headless UI on version 2 now. Giving them another try. Looks like quite a lot of people are using it now 🤔

Reply:
Good to know. It was released not too long ago with a big update to their paid package as well.